### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by rmrmrmr**
# In this capstone we:

- Added media queries for the Desktop Version of the Portfolio 🙌. This includes:
  - New Navigation Bar 🧭
  - Headline section adapted for desktop (with a new background)💻
  - My Recent Works & About Me sections are adapted for desktop. The cards adapt to the width and height of the viewport! 🧑‍💻
  - Contact Me section adapted for desktop. Also with a whole new background!📧

**NOTE: The Desktop Version was created with an ideal viewport of 1440x900 or equal aspect ratios (16:10) as provided by the Figma template. However, it adapts to the total height and width of the viewport it is displayed in. Please keep this in mind! 🤗👌**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -382,56 +379,84 @@ <h3 id="abtMeText" class="abtMeText">
...
-          </a>
-        </li>
-      </ul>
-    </div>
-    <div id="contactEmailPH">
-      <p id="contactEmail">ricardomartineza77@gmail.com</p>
+    <div id="bottomBackgroundImage">
+      <form id="contactForm" action="https://formspree.io/f/moqbvdgn" method="post">
+        <input type="text" name="name" id="nameFormBox" class="formBox1" placeholder="Full Name" maxlength="30" required>
+        <input type="email" name="email" id="mailFormBox" class="formBox1" placeholder="Email Adress" required>
```
Comment (KDlamini) [inline index.html:391]: Please make sure that your form matches the [FIGMA design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468). The [FIGMA template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)'s form has `First name` and `Last name` input fields next to each other. You only have one input field `Full name`.

Your design:

image

Expected design:

image

- Kindly fix the borders of your input fields as well. Yours are black when they should be grey as per design specifications.

image

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @rmrmrmr 🙋‍♀️,  

Great work so far! 💯 
You still need to work on some issues to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Desktop version set up correctly
✔️  Mobile-first media query is used
✔️  Good PR description
✔️  Commit messages are meaningful
✔️  No linter errors

## Required Changes ♻️

**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (rmrmrmr) [reply index.html:391]: Hi @KDlamini ! Thanks for your comments. 🙌
As per the first one, it is specified on the project requirements (https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md)  to ignore the Figma design on the two inputs issue and keep the input for the full name on every template version. 

I'll make sure to change the border colors on the next commit!

Comment (KDlamini) [reply index.html:391]: @rmrmrmr okay no problem 👍🏾

**Review by ShahierNashaat — APPROVED** [review]
Hi @rmrmrmr, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**